### ISSUE TITLE
The necessity of loss weight

### ISSUE THREAD

**Issue by WUSHUANGPPP** [mengyuest/AR-Net#5]
It's definitly a through work for action recogintion compression problem. But I still wondering why you put a different weight for acc_loss and eff_loss？ Each loss supervise each module without any overlap parameters isn't it？If it is so, then we have no necessity to set different loss weight? Appreciate it~ @mengyuest

**Comment by WUSHUANGPPP** [comment]
https://github.com/mengyuest/AR-Net/blob/2a9d72b50fed390e58cba6b3af368e3301ec078f/ops/models_ada.py#L445
Could you please tell me the impact of this function?

**Comment by mengyuest** [comment]
The different loss weights achieve a trade-off between the accuracy and efficiency preferences during the training (imagine an extreme case where you set the eff_loss=0 then the problem becomes a pure action recognition task aiming for high accuracy). Each loss is indeed supervising parameters from different modules (the accuracy loss is still influencing the policy network - otherwise, the policy network will just learn to skip every frame to reach the maximum efficiency).

The `combine_logits` function mask-select the features (from different network backbones) that want to keep/skip.
